### ISSUE TITLE
Add a hover menu to the sidebar navigation, when collapsed

### ISSUE THREAD

**Issue by kschiffer** [TheThingsNetwork/lorawan-stack#1157]
#### Summary

The sidebar menu is not sufficiently accessible when collapsed; subitems cannot be accessed while the menu is collapsed. It should provide a hover/overlay menu, showing the available sub-entries on hover (and via keyboard focus) when collapsed.

#### Why do we need this?

To enable better usage and improve accessibility and responsiveness in small screen sizes.

#### What is already there? What do you see now?

Ability to collapse the sidebar.

#### What is missing? What do you want to see?

Adding hover/overlay menus to make sub-entries accessible in this mode.

#### Environment

Console.

#### How do you propose to implement this?

Basically use the `<Dropdown />`\-component to render the full title of the entry and any possible sub-entries on hover. Just having the icons is not enough.

#### Can you do this yourself and submit a Pull Request?

Yes but help welcome @pgalic96 @mjamescompton

References #1172
